### PR TITLE
View certificate support

### PR DESCRIPTION
**Pull request by darkdh**
Currently only support on mac and windows
fix brave/browser-laptop#2611

### CODE REVIEW

**Comment by bsclifton** [comment]
Are we holding off on this one until we have a UI for displaying on Linux?

**Comment by bbondy** [comment]
No I think we should merge and build for 0.13.6 as long as we have proper handling to not use it on linux on the front end.  Please make sure an issue is posted though.

Also @darkdh this needs a rebase pls, thanks.

**Comment by bridiver** [comment]
I don't see a file for win

**Comment by darkdh** [comment]
@bridiver There is `ShowCertificateViewer` in `//chrome/browser/ui/views/certificate_viewer_win.cc`
@bbondy, rebased. and issue posted here https://github.com/brave/browser-laptop/issues/7669

**Comment by bbondy** [comment]
++

File: lib/browser/api/browser-window.js [hunk 1]
```diff
@@ -162,6 +162,9 @@ Object.assign(BrowserWindow.prototype, {
   inspectServiceWorker () {
     return this.webContents.inspectServiceWorker()
   },
+  showCertificate () {
```
Comment (bridiver) [inline lib/browser/api/browser-window.js]: is this needed for anything? There will never be a certificate for the browser-window

Comment (darkdh) [inline lib/browser/api/browser-window.js]: You are right. Removed in https://github.com/brave/muon/commit/4deb4b2bfa515889c69bd9e44aa7d80c858dad73

**Pull request merged**